controllers/operator.go

```diff
@@ -15,3 +15,37 @@ limitations under the License.
 */
 
 package ocibuilder
+
+import (
+	"github.com/ocibuilder/ocibuilder/common"
+	"github.com/ocibuilder/ocibuilder/pkg/apis/ocibuilder/v1alpha1"
+	"github.com/sirupsen/logrus"
+)
+
+// the context of an operation on a ocibuilder object.
+// the controller creates this context each time it picks a ocibuilder object off its queue.
+type operationContext struct {
+	// builder is the ocibuilder object
+	builder *v1alpha1.OCIBuilder
+	// updated indicates whether the ocibuilder object was updated and needs to be persisted back to k8
+	updated bool
```

Comment:
This is giving a linting error for being unused

Reply:
hmm, its gonna get used later in the code. I can comment it for now.

Reply:
cool

Reply:
let me check if updated: false in init struct satisfies lint.

Reply:
I don't like that the CI lint didnt understand the default value for bool and complained unnecessarily.